tests/resolvers/ChatMessage/replyTo.spec.ts

```diff
@@ -0,0 +1,95 @@
+import "dotenv/config";
+import { replyTo as replyToResolver } from "../../../src/resolvers/ChatMessage/replyTo";
+import { connect, disconnect } from "../../helpers/db";
+import type mongoose from "mongoose";
+import { ChatMessage } from "../../../src/models";
+import { beforeAll, afterAll, describe, it, expect, vi } from "vitest";
+import type { TestChatMessageType } from "../../helpers/chat";
+import {
+  createTestChatMessageWithoutReply,
+  createTestChatMessage,
+} from "../../helpers/chat";
+import { Types } from "mongoose";
+import { MESSAGE_NOT_FOUND_ERROR } from "../../../src/constants";
+
+let testChatMessage: TestChatMessageType;
+let testChatMessageWithoutReply: TestChatMessageType;
+let MONGOOSE_INSTANCE: typeof mongoose;
+
+beforeAll(async () => {
+  MONGOOSE_INSTANCE = await connect();
+  const temp = await createTestChatMessageWithoutReply();
+  const temp1 = await createTestChatMessage();
+  testChatMessageWithoutReply = temp[3];
+  testChatMessage = temp1[3];
+});
+
+afterAll(async () => {
+  await disconnect(MONGOOSE_INSTANCE);
+});
+
+describe("resolvers -> DirectChatMessage -> replyTo", () => {
+  it(`returns directChat object for parent.replyTo`, async () => {
```

Comment:
If `replyToResolver` or `ChatMessage.findOne` fail in the first test, it may cause errors that aren’t caught.  Add explicit try-catch blocks around async calls within your test cases, especially for the first test. 

```javascript
it(`returns directChat object for parent.replyTo`, async () => {
  const parent = testChatMessage?.toObject();
  if (!parent) throw new Error("Parent object is undefined.");
  if (typeof replyToResolver !== "function") throw new Error("replyToResolver is not a function.");

  try {
    const replyToPayload = await replyToResolver(parent, {}, {});
    const replyTo = await ChatMessage.findOne({ _id: testChatMessage?.replyTo }).lean();
    expect(replyToPayload).toEqual(replyTo);
  } catch (error) {
    console.error("Test failed", error);
    throw error;
  }
});
```